test/actions/create.spec.js

```diff
@@ -202,7 +202,7 @@ describe('Create action creator', () => {
       headers: {
         'Content-Type': 'application/vnd.api+json',
       },
-      endpoint: 'api.test/{param1}/{param2}/abc',
+      endpoint: 'api.test/{param1}/{param2}/abc{?q1,q2}',
```

Comment:
Idea was to test will urijs resolve query parameters that are passed as `q1`, `q2`